src/modules/core/components/Tag/Tag.css

```diff
@@ -1,8 +1,9 @@
 .baseStyles {
-  padding: 1px 10px 2px;
+  padding: 1px 8px 2px;
   border-radius: var(--radius-large);
   font-size: 0.8rem;
   font-weight: var(--weight-bold);
+  color: var(--colony-white);
 }
```

Comment:
This is not something that you added, but something I've noticed:

![Screenshot from 2021-03-12 11-28-12](https://user-images.githubusercontent.com/1193222/110920762-72072980-8326-11eb-996c-7a9fb592bc72.png)

Tags are broken up at the end of the line. I don't that this should be the case.

Please add either something like `display: inline-block` or `white-space: nowrap` to prevent that from happening.